pkg/core/namespace.go

```diff
@@ -248,3 +285,96 @@ func readPassword(s string) (string, error) {
 		return string(res), err
 	}
 }
+
+func getInitLabels() map[string]string {
+	return map[string]string{"created-by": fmt.Sprintf("%s-%s", env.Cli.Name, env.Cli.Version)}
```

Comment:
I'd like to understand the intention here. Which versions of riff should be able to cleanup namespaces created by which other versions of riff?

Reply:
That's a good question and I'm not sure what we want to do here.

Reply:
Unless `created-by` is a conventional label, which doesn't appear to be the case, we could change this label to, for example, `"projectriff.io/riff-version":"0.3.0-snapshot"`. I'm suggesting using a prefix in the key to avoid clashing with user-defined labels.

I would suggest that we keep the semantics simple at first and ignore the value of the version in the label, so any version of riff could cleanup the resources of any other version. But having the version in the label is informative for users and would allow us to take into account the version if that becomes necessary.

Reply:
Shouldn't these be two separate labels? Who installed the resource is relevant during cleanup. The version of the resource is relevant during upgrade.